test/banach/retry_test.clj

```diff
@@ -5,6 +5,40 @@
             [clojure.math.numeric-tower :as math]
             [banach.retry :as retry]))
 
+(def ctx-1 {:failures [(Exception. "Slow your roll")]})
+(def ctx-2 (update ctx-1 :failures conj (Exception. "KABOOM")))
+
+(deftest routing-tests
+  (let [strat (retry/routing
+               (comp odd? count :failures) identity
+               (constantly true) retry/give-up)]
```

Comment:
Yep, that is correct. Do you feel the most confusing part is:

1. the predicate
2. `identity` as a strategy
3. something else?